x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/select_inference_id.tsx

```diff
@@ -89,13 +89,15 @@ const SelectInferenceIdContent: React.FC<SelectInferenceIdContentProps> = ({
   value,
 }) => {
   const {
-    core: { application, http },
+    core: { application },
     docLinks,
-    plugins: { ml },
+    plugins: { ml, share },
   } = useAppContext();
   const config = getFieldConfig('inference_id');
 
-  const inferenceEndpointsPageLink = `${http.basePath.get()}/app/enterprise_search/relevance/inference_endpoints`;
+  const inferenceEndpointsPageLink = share?.url.locators
```

Comment:
I'm new to this code - mind posting a screenshot that shows the UI affected by this? Also, can you remind me why this conditional is needed and which environments lack the enterprise_search app? I can assume, but you might know better than me and it would be nice to have it directly mentioned in the pr

Reply:
Hi @mattkime , Thank you for bringing this up. We are releasing `inference management UI` in Serverless (PR title fixed) and hit many small issues that needed fixing.

This particular code change required as `Manage Inference Endpoint` in Popover links(please see attached screenshot) were hardcoded and will always redirect to enterprise search -> inference endpoints. However, for Serverless, we want them to redirect to Serverless -> inference endpoints.

This is a bit odd as both `enterprise-search` and `serverless` render the same plugin into two different URLs. From `81.8`, this will change however we need a fix till then.

Hope that helps.

![Screenshot 2024-11-21 at 11 49 34 AM](https://github.com/user-attachments/assets/0c42c1a2-76d5-4dbf-a8ba-d6db4d42e634)